src/Uno.SourceGeneration.Host/Uno.SourceGeneration.Host.csproj

```diff
@@ -67,7 +67,7 @@
       <Version>1.3.1</Version>
     </PackageReference>
     <PackageReference Include="Umbrella">
-      <Version>2.41.1</Version>
+      <Version>2.42.1-dev.2824</Version>
```

Comment:
why did it update to -dev?

Reply:
@jeremiethibeault @jeromelaban why does Uno.sourceGeneration reference Umbrella, it won't compile for everyone as the package is not public right?

Reply:
From the looks of it, this project is not used at all. It's not referenced in the solution nor in any other projects. Unless there is some magic being done we could safely remove this project. @jeromelaban  could confirm.